### DISCUSSION TITLE
Adding webpack plugin to tina config to fix "could not resolve path | os | crypto" error

### DISCUSSION THREAD

**Discussion by haowens** [tinacms/tinacms#4093] [Help] [answered]
Hi! I was wondering if anyone knew how to go about resolving this error that comes about when I try to "npm run dev"? The dotenv docs say the solution is to add a webpack plugin, but i've tried installing a couple and configuring them in my next.config.js and can't seem to get anything to work.
<img width="796" alt="Screenshot 2023-07-17 at 10 35 15 p m" src="https://github.com/tinacms/tinacms/assets/69762131/9a63c86d-823a-4d63-b009-ccd53eadc033">

**Comment by kldavis4** [comment]
What version of tinacms & @tinacms/cli?

**Reply by haowens** [reply]
"tinacms": "1.5.11", "@tinacms/cli": "1.5.20"

**Reply by kldavis4** [reply, marked as answer]
can you update to the latest of both those dependencies and retry?

**Reply by haowens** [reply]
fixed!!! thank you
